### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maxbertinetti**
Removed two obsolete implementations for VS-Code
Added Geany Editor
Moved LSP and added Crystalline (https://github.com/elbywan/crystalline)

## Checklist
* [X] - Shard is at least 30 days old.
* [X] - Shard has CI implemented.
* [X] - Shard has daily/weekly periodic builds (ideally with Crystal latest and nightly).

### CODE REVIEW

**Comment by maxbertinetti** [comment]
I want to request the rights to review for helping with this project

**Review by veelenga — APPROVED** [review]
@maxbertinetti Looking good. Please fix the specs.

> I want to request the rights to review for helping with this project

Thanks, i've just sent you an invite.

**Comment by maxbertinetti** [comment]
I think I cannot review my PR.....

File: README.md [hunk 1]
```diff
@@ -562,9 +563,11 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
    * [vim-crystal](https://github.com/vim-crystal/vim-crystal) - Vim filetype support for Crystal
    * [vim-slang](https://github.com/elorest/vim-slang) - Vim filetype support for Slang Templating Engine
  * Visual Studio Code
-   * [vscode-crystal](https://github.com/g3ortega/vscode-crystal) - Crystal language support in VSCode
-   * [vscode-crystal-ide](https://github.com/crystal-lang-tools/crystal-ide) - Crystal IDE powered by [Language Server Protocol](https://code.visualstudio.com/blogs/2016/06/27/common-language-protocol)
    * [vscode-crystal-lang](https://github.com/crystal-lang-tools/vscode-crystal-lang) - Formatter, linter and syntax highlighting for `cr` and `ecr` files
+   
+## LSP Implementations
```
Comment (Blacksmoke16) [inline README.md:568]: This should be added to the TOC.

Comment (maxbertinetti) [reply README.md]: You are right. Done!

**Comment by maxbertinetti** [comment]
@Blacksmoke16 test passes and toc adjusted.

**Comment by veelenga** [comment]
Thanks

**Pull request merged**